Assets/MixedRealityToolkit/Extensions/EditorClassExtensions/ScriptableObjectExtensions.cs

```diff
@@ -67,5 +67,25 @@ public static ScriptableObject CreateAsset(this ScriptableObject scriptableObjec
 
             return instances;
         }
+
+        /// <summary>
+        /// Gets all the scriptable object instances in the project.
+        /// </summary>
+        /// <param name="profileType">The Type of <see href="https://docs.unity3d.com/ScriptReference/ScriptableObject.html">ScriptableObject</see> you're wanting to find instances of.</typeparam></param>
+        /// <returns></returns>
+        public static ScriptableObject[] GetAllInstances(Type profileType)
```

Comment:
add where T : BaseMixedRealityProfile (or whatever the class)

Reply:
This isn't a generic method.

Reply:
That works but then rename argument to be assetType or typeFilter etc not profile type